cmd/snap-preseed/main_test.go

```diff
@@ -373,6 +379,7 @@ func (s *startPreseedSuite) TestClassicRequired(c *C) {
 func (s *startPreseedSuite) TestRunPreseedUnsupportedVersion(c *C) {
```

Comment:
shouldn't we have a TestRunPreseedDebIsNewer as well?

Reply:
Added.